src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -1956,6 +1968,516 @@ public Task OpenAsync(CancellationToken token)
             }
         }
 
+        private class ServerSecurityDuplexSessionChannel : ServerSecuritySessionChannel //, IDuplexSessionChannel 
+        {
+            SoapSecurityServerDuplexSession session;
+            bool isInputClosed;
```

Comment:
Missing accessors. Naming convention is wrong. And remove `this.` in the code of this class